ui/static/ui/css/slide-drawer.css

```diff
@@ -81,6 +81,7 @@ html, body {
   -webkit-transition: visibility 0s 0s;
   -moz-transition: visibility 0s 0s;
   transition: visibility 0s 0s;
+  overflow: scroll;
```

Comment:
This leaves a horizontal scrollbar at the bottom of the page after the panel closes. What is this for?